BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -466,4 +513,35 @@ open class BraintreeClient @VisibleForTesting internal constructor(
     open fun launchesBrowserSwitchAsNewTask(launchesBrowserSwitchAsNewTask: Boolean) {
         this.launchesBrowserSwitchAsNewTask = launchesBrowserSwitchAsNewTask
     }
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    fun getGraphQLMutationName(jsonString: String): String {
```

Comment:
I think we can remove this method now since we're getting the query name directly from the JSON.

Reply:
Thank you, removed here: [6d06cfe](https://github.com/braintree/braintree_android/pull/1032/commits/6d06cfeded1574f3c06e6d8c2104d261f224a297)